SPIKES.md

```diff
@@ -68,11 +76,6 @@ instead be created as regular issues.
 - Spikes can be created from existing issues. Bug reports are a good example of this. When converting
   existing issues, make sure the issues have all of the requirements from above.
 - If a spike only has a single deliverable and a low timebox, consider adding it as a task to another
-  related issue
-
-### How do I complete a spike?
-
-Determining whether a spike is complete should be an easy task provided
-that the deliverables have been clearly defined. When this is not the case, you should
-not work on the issue and instead attempt to clarify the deliverables with those also
-working on the project.
+  related issue.
+- Spikes need deliverables. If there are no deliverables listed, do not work on the spike and instead
+  determine what these deliverables are before going forward and update the spike.
```

Comment:
```suggestion
- Spikes need deliverables. If there are no deliverables listed, do not work on the spike and instead
  determine what these deliverables are before going forward and updating the spike.
```

Reply:
@kathatherine, here I'm going to stick with what I wrote because I mean it like:

Do not work on the spike and instead:
- determine ....
- update the spike

Hope that makes sense 😄

Reply:
With your clarification in mind, I suggest rewording this sentence like:

```
Spikes need deliverables. If there are no deliverables listed, do not work on the spike; instead, determine what these deliverables are and then update the spike with that information.
```

Reply:
I agree with Bianca's edit.

Reply:
Thanks @beeankha!

```suggestion
- Spikes need deliverables. If there are no deliverables listed, do not work on the spike; instead, determine what these deliverables are and then update the spike with that information.
```